docs/snippets/all/archetypes/scalar_send_columns.rs

```diff
@@ -1,24 +1,23 @@
 //! Use the `send_columns` API to send scalars over time in a single call.
 
-use rerun::components::Scalar;
 use rerun::TimeColumn;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let rec = rerun::RecordingStreamBuilder::new("rerun_example_send_columns").spawn()?;
+    let rec = rerun::RecordingStreamBuilder::new("rerun_example_scalar_send_columns").spawn()?;
 
-    // Native time / scalars
-    let timeline_values = (0..64).collect::<Vec<_>>();
+    const STEPS: i64 = 64;
 
-    let scalar_data: Vec<f64> = timeline_values
-        .iter()
-        .map(|step| (*step as f64 / 10.0).sin())
-        .collect();
+    let times = TimeColumn::new_sequence("step", 0..STEPS);
+    let scalars = (0..STEPS).map(|step| (step as f64 / 10.0).sin());
 
-    // Convert to rerun time / scalars
-    let timeline_values = TimeColumn::new_sequence("step", timeline_values);
-    let scalar_data: Vec<Scalar> = scalar_data.into_iter().map(Into::into).collect();
-
-    rec.send_columns("scalars", [timeline_values], [&scalar_data as _])?;
+    rec.send_columns_v2(
+        "scalars",
+        [times],
+        rerun::Scalar::update_fields()
+            .with_many_scalar(scalars)
+            .columns(std::iter::repeat(1).take(STEPS as _))?
+            .filter(|column| !column.descriptor.component_name.contains("Indicator")),
```

Comment:
The other languages do not log an indicator here, and for good reasons: indicators in a scalar chunk have huge overhead, and are completely useless as far as I can tell (i.e. they aren't needed for heuristics).

In a follow-up, I will introduce a hardcoded exception so that `Scalar::columns` just doesn't generate it.
* See also #8768